OrbitGl/CaptureWindow.cpp

```diff
@@ -899,9 +899,31 @@ void CaptureWindow::RenderHelpUi() {
 }
 
 //-----------------------------------------------------------------------------
-ImTextureID TextureId(uint32_t id) {
-  uint64_t texture_id = id;
-  return reinterpret_cast<ImTextureID>(texture_id);
+ImTextureID TextureId(uint64_t id) { return reinterpret_cast<ImTextureID>(id); }
```

Comment:
nit: since ImTextureId is void* you probably want to do reinterpret_cast<ImTextureID>(static_cast<uintptr_t>(id)) to ensure the integer is of sizeof(void*) on all platforms.

Reply:
Thanks Dimitry, I integrated your suggestion.